app/global.R

```diff
@@ -87,25 +87,21 @@ conditions_list = c("Chronic renal disease" = "renal_disease",
                     "Other chronic condition" = "other")
 
 # activities list placebolder 
-l_activities_list = c("grocery shopping", 
-                      "library or museum",
-                      "outdoor socially distanced exercise (e.g. tennis, golf, bike riding)", 
-                      "outdoor visit with friends")
-m_activities_list = c("children’s playdate and playgrounds",
-                      "indoor public spaces (e.g. salon, casino, bowling, shopping mall)",
-                      "public beach or pool",
-                      "public transportation/airplane",
-                      "overnight stay at a hotel", 
-                      "outdoor dinning",
-                      "see doctor or dentist", 
-                      "school, camp, or daycare",
-                      "working outside of home")
-h_activities_list = c("indoor restaurant or bar", 
-                      "nightclub/music concert/movie theater/gym",
-                      "sports stadium",
-                      "religious services (including weddings and funerals)", 
-                      "team sports (basket ball, football)", 
-                      "visiting nursing home or hospital")
+l_activities_list = c("Grocery shopping", 
+                      "Visiting library or museum",
+                      "Outdoor socially distanced activities")
+m_activities_list = c("Indoor public spaces (e.g. salon, casino, bowling, shopping mall)",
```

Comment:
Do you think going to a regular store (like a clothing store or craft store) fits here, or lower risk with grocery shopping, library, and museum.  Maybe we can try to clarify this since this is a common activity. Does "indoor public spaces" mean "crowded indoor spaces"?

Reply:
Do you think shopping mall encompasses a regular store? I assumed that when people see mall, they would also think of stand alone stores. Do you have a suggestion for a place name that indicates a regular store?

Good question about grocery shopping, libraries, and museums. I like your idea of crowded vs not crowed indoor spaces but I am not sure if it capture's BellAge's intention. I am not sure why BellAge classifies the grocery shopping differently than than shopping malls and if the intended distinction is the level of crowdedness since grocery stores could be more crowed than a shopping mall. What do you think?

Reply:
Hi both, thanks for your discussion. My understanding is BellAge took the data published by "Information is Beautiful". Here is the link to the original webpage where they explained more of their methodology. https://informationisbeautiful.net/2020/coronavirus-covid-19-riskiest-activities/
The datasheet can be found here: https://docs.google.com/spreadsheets/d/1g_YxmDfQx7aOU2DKzNZo9b-NTk62Bju6X3z6OuCa6gw/edit#gid=1210847894

Reply:
Ah cool, maybe it is best to just stick with their original wording to avoid making incorrect assumptions. By the way, did we draw the line between medium and high risk at a different place than the Bellage scale? (Some activities, like outdorr dining, are categorized as high risk by us but at the high end of medium risk by Bellage)

Reply:
Thank you Cindy for the links. They are very helpful. Based on the "notes" column of the data sheet, it looks like the main concern about activities are: how crowed and well ventilated are places. Maybe we could have "social distanced indoor spaces (e.g. grocery stores)" and "busy or crowed indoor spaces (i.e. casino, shopping mall, salon)"